clusterloader2/docs/experiments.md

```diff
@@ -0,0 +1,44 @@
+# clusterloader2 - experiment rollout
+
+In this doc any change to the behavior of clusterloader2 is referred as an "experiment".
+
+## Motivation
+
+clusterloader2 is a tool used a by all scalability (performance) tests. Tests compile clusterloader2 at [HEAD](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L121). Introducing breaking changes to clusterloader2 will stop tests from passing at all. While for smaller and faster tests, this is not a huge costs (unless this happens on the weekend, see https://github.com/kubernetes/perf-tests/pull/586), this can be costly in rarely run, expensive tests (e.g. [ci-kubernetes-e2e-gce-scale-performance](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-scalability/sig-scalability-release-blocking-jobs.yaml#L44)).
+
+For this reason, all new changes features added to clusterloader2, shall be gated and rolled out gradually. This way we can minimize blast radius of breaking changes. We can even remove some kinds of them to happen at all, as they should be caught at presubmit time and not allowed to merge at all.
+
+## Experiment rollout
+
+### General principles
+
+1. If for some reason it doesn't make sense to enable an experiment in particular test job indicate the reason clearly in the PR (e.g. measuring kube-proxy metrics in kubemark can be skipped as kube-proxy is not running there).
```

Comment:
NIT: It's not true that kube-proxy doesn't run there, a hollow kube-proxy runs there

Reply:
TIL! 
However I still expect it to not do real work, so it doesn't make sense to scrape it in this case. I'll rephrase this sentences.